pages/linux/sattach.md

```diff
@@ -0,0 +1,16 @@
+# sattach
+
+> Attach to a Slurm job step.
+> More information: <https://slurm.schedmd.com/sattach.html>.
+
+- Make available the IO streams (`stdout`, `stderr`, and `stdin`) of a Slurm job step:
```

Comment:
Define `Make available` here.

Reply:
```suggestion
-  Redirect the IO streams (`stdout`, `stderr`, and `stdin`) of a Slurm job step to the current terminal:
```

Do you think this would be better?

Reply:
Yes, that looks much better @gutjuri. Many thanks!